web-registry/src/components/organisms/SellOrdersTable/SellOrdersTable.Row.tsx

```diff
@@ -73,18 +73,18 @@ const getSellOrdersTableRow = ({
     {batchDenom}
   </Link>,
   <Box sx={{ display: 'flex', alignItems: 'center' }}>
-    {!disableAutoRetire && (
+    {disableAutoRetire && (
       <InfoLabel
-        label="Tradeable"
+        label="Tradable"
         variant="success"
         icon={<SellOrderPurchaseIcon icon="arrowDown" />}
         sx={{ mr: 4 }}
       />
     )}
     <InfoLabel
-      label="Retired"
+      label="Retirable"
       variant="default"
-      icon={<SellOrderPurchaseIcon icon="tradeable" />}
+      icon={<SellOrderPurchaseIcon icon="tradable" />}
```

Comment:
definitely just a nit, but it seems a little funny that the label is "retirable" but the icon is "tradable"..

Reply:
Actually, this was an issue, thanks for catching this!